### PR TITLE
Update Prelude.dhall to v10.0.0

### PR DESCRIPTION
**Pull request by sjakobi**
Now all the examples will build again with current versions
of dhall.

### CODE REVIEW

**Comment by sjakobi** [comment]
Thank you for the invitation @f-f! :)

Should I wait for an explicit approval, or is this trivial enough that I can simply merge?

**Review by f-f — APPROVED** [review]
Thanks! You now have write access so you can merge yourself 🙂

**Pull request merged**
